images/node/.gitignore

```diff
@@ -0,0 +1,4 @@
+bin/
```

Comment:
We don't have to do this now but it would be better to have one subdirectory ("content") that everything is under and ignored, rather than having to add one directory each time.  Can you open a follow up issue to track that?